models/wrf_hydro/noah_hydro_mod.f90

```diff
@@ -982,6 +981,7 @@ subroutine get_routelink_constants(filename)
 integer, allocatable :: fromIndsStart(:)
 integer, allocatable :: fromIndsEnd(:)
 integer, allocatable :: toIndex(:)
+!integer, allocatable :: num_up_links(:)
```

Comment:
num_up_links is now a module global, so better to delete this line rather than comment it out. 
```suggestion
```

Reply:
Done